ui/index.html

```diff
@@ -21,11 +27,13 @@
 <table class="tg">
     <tr>
         <th class="tg-031e">Stream</th>
+        <!--InjectHeaders-->
         <th class="tg-yw4l"></th>
         <th class="tg-yw4l"></th>
+        <th class="tg-yw4l"><button onclick="resetAllStreams('http://localhost:9191/master/apple/bipbopall.m3u8?resetStream=1')">Reset</button></th>
```

Comment:
If you want to reset all streams (even ones that aren't in the master playlist), you'll want to send in a value of resetStream=2.

Also, you probably want to avoid sending in a hardcoded value of localhost, since we want to be able to account for situations where Socrates isn't on your local machine.